.github/workflows/release.yml

```diff
@@ -26,11 +26,28 @@ jobs:
         uses: snok/install-poetry@v1.3.3
       - name: configure pypi credentials
         run: poetry config pypi-token.pypi "${{ secrets.PYPI_API_KEY }}"
-      - name: Publish package
+      - name: Publish core package
         run: poetry publish --build
       - name: install dependencies
         run: poetry install
       - name: update metadata
         env:
           DATABASE_URL: ${{ secrets.OPENSTATES_DATABASE_URL }}
         run: poetry run os-initdb
+      # Add steps for openstates-metadata package
+      - name: Create directory and copy openstates-metadata
+        run: |
+          mkdir -p /openstates-core-metadata/openstates-metadata
+          cp -r openstates/metadata /openstates-core-metadata/openstates-metadata
+      - name: Change directory to openstates-metadata
+        run: cd /openstates-core-metadata/openstates-metadata
+      - name: Move configuration files
+        run: |
+          mv pyproject.toml.sample pyproject.toml
+          mv .python-version.sample .python-version
```

Comment:
seems like we also need a `mv` for `README.md.sample`

Reply:
Ah I see my changes from last night were not committed, thanks.